experiment/process-e2e-suite-log/README.md

```diff
@@ -0,0 +1,63 @@
+# target 
+To process the input file and extract only a given selection of visited endpoints.
+
+## "input file":
+it is a "master log," that is the collection of the endpoints visited while executing a testing run.  
+Here is the [input][input source]: 
+
+This file has an expected and known format: 
+- line 1: "separatorLine".
+- line 2: timestamp.
```

Comment:
What is the format of the timestamp?

Reply:
Complete date plus hours, minutes, seconds with fractions.
Example: 2018-06-05T03:34:03.723486

(I am currently not using this field in this analysis, but describing what was expected in the file by that time)